src/components/CommonCard/styles.less

```diff
@@ -0,0 +1,12 @@
+@import '@assets/theme/constants.less';
+
+.common-card-wrapper {
+  border: 1px solid #070a26;
```

Comment:
I had seen the same color in other style file

Reply:
It will be improved in the next version.